web/src/app.css

```diff
@@ -136,4 +137,12 @@ input:focus-visible {
   .scrollbar-stable {
     scrollbar-gutter: stable both-edges;
   }
-}
+
+  [data-color-scheme="dark"] {
```

Comment:
These aren't really utilities, so I would move them out of this layer. The `dark` class gets already applied and we might as well use that instead of adding a data property:

```css
:root.dark {
  color-scheme: dark;
}

.root:not(.dark) {
  color-scheme: light;
}
```

Reply:
Sorry made a typo here

```diff
- .root:not(.dark) {
+ :root:not(.dark) {
```

Reply:
i don't see it either but it looks like it just needs `:root.dark`?